editors/jetbrains/README.md

```diff
@@ -1,16 +1,27 @@
-### Configure External Tool
+## Configure External Tool
 
-Go to *File | Settings | Tools | External Tools* for Windows and Linux or *WebStorm | Preferences | Tools | External Tools* for OS X and click **+** to add a new tool. Let’s name it **Prettify**.
+https://blog.jetbrains.com/webstorm/2016/08/using-external-tools/
 
-In **Program** set `prettier`
+Go to *File | Settings | Tools | External Tools* for Windows and Linux or *WebStorm | Preferences | Tools | External Tools* for OS X and click **+** to add a new tool. Let’s name it **Prettier**.
 
-In **Parameters** set `--write [other opts] $FilePathRelativeToProjectRoot$` 
+* **Program** set `prettier`
 
-In **Working directory** set `$ProjectFileDir$`
+> If on the other hand you have `prettier` installed locally, replace the **Program** with `./node_modules/.bin/prettier` (on OS X and Linux) or `.\node_modules\.bin\prettier.cmd` (on Windows).
+
+* **Parameters** set `--write [other opts] $FilePathRelativeToProjectRoot$` 
+* **Working directory** set `$ProjectFileDir$`
 
 ![Example](https://raw.githubusercontent.com/jlongster/prettier/master/editors/jetbrains/with-prettier.png)
 
-If on the other hand you have `prettier` installed locally, replace the **Program** with `./node_modules/.bin/prettier` (on OS X and Linux) or `.\node_modules\.bin\prettier.cmd` (on Windows).
+### Process directories
+
+* Clone the External tool and name it `Prettier Directories`
+* **Parameters** set `--write [other opts] $FilePathRelativeToProjectRoot$/**/(*.js|*.jsx)` 
```

Comment:
> Not sure what the first comment is implying. I just kept the documentation the same.

This template `--write $FilePathRelativeToProjectRoot$/**/(*.js|*.jsx)`
Generates cmd like this `./node_modules/.bin/prettier --write /path/to/file.js/**/(*.js|*.jsx)`
If you run it against file.
>  It is nice to be able to select a directory in IDEA and have it format all the files in the directory with a couple keypresses.

If you want to format whole directory - it's better to create separate "External Tool" to deeply format files in it or write custom script - which will detect ether it's file or directory and pass parameters to prettier.